tensorflow/compiler/xla/service/algebraic_simplifier.cc

```diff
@@ -3283,6 +3360,31 @@ Status AlgebraicSimplifierVisitor::HandleSlice(HloInstruction* slice) {
   return Status::OK();
 }
 
+Status AlgebraicSimplifierVisitor::HandleRsqrt(HloInstruction* rsqrt) {
+  VLOG(10) << "trying transform [rsqrt(Pow(A, -2)) => |A|] "
+           << rsqrt->ToString();
+  HloInstruction* rsqrt_operand = rsqrt->mutable_operand(0);
```

Comment:
`rsqrt_operand` here is not `A` but `Pow`. Do you want me to change `rsqrt_operand` to something like `pow`? But in that case, it might be misleading since `rsqrt`'s operand may or may not be `Pow`. Sorry if I misunderstood your comment.

Reply:
I mis-read the code, rsqrt_operand is fine.